operators.js

```diff
@@ -24,10 +24,11 @@ const fromEvent = (target, eventName) => {
  */
 const interval = (ms) => {
     let _intervalId
+    let i = 0
     return new ReadableStream({
         start(controller) {
             _intervalId = setInterval(() => {
-                controller.enqueue(Date.now())
+                controller.enqueue(i++, Date.now())
```

Comment:
add i++ param, like the interval https://www.learnrxjs.io/learn-rxjs/operators/creation/interval